README.adoc

```diff
@@ -312,9 +339,23 @@ To add configuration to the `inventory` service, the [hotspot=config-injection f
 The port number from the configuration is retrieved by the [hotspot=config-port file=0]`inventoryConfig.getPortNumber()` method and passed to the [hotspot=config-port file=0]`manager.get()` method as a parameter.
 
 To determine whether the inventory service is in maintenance or not (according to the configuration value), [hotspot=isInMaintenance file=0]`inventoryConfig.isInMaintenance()` class method is used.
-If you set the [hotspot=4 file=1]`io_openliberty_guides_inventory_inMaintenance` property to `true` in the configuration, the inventory service returns the message, `ERROR: Service is currently in maintenance`, along with the contact email.
+If you set the [hotspot=4 file=2]`io_openliberty_guides_inventory_inMaintenance` property to `true` in the configuration, the inventory service returns the message, `ERROR: Service is currently in maintenance`, along with the contact email.
 The email configuration value can be obtained by calling [hotspot=getEmail file=0]`inventoryConfig.getEmail()` method.
 
+
+Let's use the externalized configuration in the `system` service as well.
```

Comment:
We should try to avoid first person pronouns in our guides according to our [style guide](https://github.com/OpenLiberty/draft-guides-template/wiki/Guidelines-for-Structure-and-Styles#language), so we should try to rewrite this sentence.

Reply:
Thanks for the feedback and resource!